G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/WordGetText/ShoudReturnText_WhenGetTextCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,5 @@
+﻿word.open
+word.inserttext text ‴Test text.‴
+word.gettext result ♥text
+word.close
+test ⊂♥text.ToString().Trim().Equals("Test text.")⊃
```

Comment:
Don't use `Equals()`, it's not freakin' java 🤪
This issue applies to other tests as well.